resources/views/themes/default1/front/clients/show-order.blade.php

```diff
@@ -125,75 +118,84 @@
                                                     <td>
                                                         @foreach($installationDetails['installed_ip'] as $paths)
-                                                        <li>{{$paths}}</li>
+                                                       {{$paths}}<br>
                                                         @endforeach
                                                     </td>
                                                     @else
                                                      <td>
                                                     --
-                                                  </td>
+                                                    </td>
                                                   @endif
+                                                  <td></td>
                                                 </tr>
                                                   
 
                                                 <?php
-
-                                                if (!$subscription || strtotime($subscription->ends_at) < 1) {
-                                                    $sub = "--";
-                                                } else {
-                                                    $date = new DateTime($subscription->ends_at);
-                                                    $tz = \Auth::user()->timezone()->first()->name;
-                                                    $date->setTimezone(new DateTimeZone($tz));
-                                                      
-                                                    $sub = $date->format('M j, Y, g:i a ');
-                                                    // $sub = $sub2->setTimezone($tz);
-                                                }
-
-                                                if (!$subscription || strtotime($subscription->update_ends_at) < 1) {
-                                                    $update_sub = "--";
-                                                } else {
-                                                    $date1 = new DateTime($subscription->update_ends_at);
-                                                    $tz = \Auth::user()->timezone()->first()->name;
-                                                    $date1->setTimezone(new DateTimeZone($tz));
-                                                      
-                                                    $update_sub = $date1->format('M j, Y, g:i a ');
-                                                    // $sub = $sub2->setTimezone($tz);
+                                                $date= '--';
+                                                $licdate = '--';
+                                                $supdate = '--';
+                                                if ($subscription) {
```

Comment:
this logic was present even in the controller
why here again?

Reply:
That was for admin Panel .Made similar changes for client panel also.